python/dgl/heterograph.py

```diff
@@ -304,6 +335,24 @@ def add_edges(self, u, v, data=None, etype=None):
     # Metagraph query
     #################################################################
 
+    @property
+    def is_unibipartite(self):
+        """Return whether the graph is a uni-bipartite graph.
```

Comment:
We may want to reconsider how we call such graph structures.  At least "unidirectional bipartite graph" refers to the graphs with only one edge type and two node types in our docstrings.

Reply:
I think uni-directional bipartite graph is the word for this kind. More formally, given the definition of uni-directional bipartite graph for homogeneous graph, we can extend this definition to heterogeneous graph by considering the metagraph -- a heterograph is uni-directional bipartite iff its metagraph is uni-directional bipartite.

A block graph is a special uni-bipartite graph with nodes properly relabeled and each DST node has one mirror in the SRC group.